lib/modules/platform/github/types.ts

```diff
@@ -54,7 +54,8 @@ export interface GhRestPr {
 }
 
 export interface GhPr extends Pr {
-  updatedAt: string;
+  updated_at: string;
+  node_id: string;
```

Comment:
Will be refactored to camel case later. For now, it's snake case for a good reason.